Makefile

```diff
@@ -1,3 +1,10 @@
+TARGET_SYS?=arm-linux-gnueabihf
```

Comment:
So I would leave the default target probably an empty string (with a comment what to set it to for cross-compilation (or the environment variable)). Then below making sure that the if-statement below is not executed when $(TARGET_SYS) is an empty string.